### PR TITLE
CleanBuild for role_roles

### PR DESCRIPTION
**Pull request by saargt**

### CODE REVIEW

File: server-side/services/builders/build.service.ts [hunk 1]
```diff
@@ -16,9 +17,9 @@ export class BuildService
...
 	}
 
-	public async buildAdalTable(body: any): Promise<any>
+	public async buildAdalTable(body: any): Promise<AsyncResultObject>
 	{
-    	const res = { success: true };
+    	const res: AsyncResultObject = { success: true };
     	try
     	{
 			await this.hideAdalItems();
```
Comment (ShirMarciano) [inline server-side/services/builders/build.service.ts:25]: I missed that
it happen on users and account_users also??

Comment (saargt) [reply server-side/services/builders/build.service.ts:25]: AFAIK, only role_roles has a 'true' value for this.buildServiceParams.shouldCleanBuild (this.hideAdalItems(); checks this value. It performs a clean build only if it is true.)
@leonPep42  am I right?

Comment (ShirMarciano) [reply server-side/services/builders/build.service.ts:25]: flag? :)

Comment (saargt) [reply server-side/services/builders/build.service.ts:25]: A guard, not a flag! :)

**Pull request merged**